pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Produces fake instrument data for testing.
+"""
+
+import pandas as pds
+import numpy as np
+import pysat
+
+platform = 'pysat'
+name = 'testing2D_xarray'
+
+pandas_format = False
+
+def init(self):
+    self.new_thing=True        
+
+
+def load(fnames, tag=None, sat_id=None):
+    import xarray as xr
+    
+    # create an artifical satellite data set
+    parts = fnames[0].split('/')
+    yr = int('20'+parts[-1][0:2])
+    month = int(parts[-3])
+    day = int(parts[-2])
+    date = pysat.datetime(yr,month,day)
+    # scalar divisor below used to reduce the number of time samples
+    # covered by the simulation per day. The higher the number the lower
+    # the number of samples (86400/scalar)
+    scalar = 1
+    num = 86400/scalar
```

Comment:
You don't have future division here, so this scalar division will behave differently for python 2 and 3, depending on the value of 'scalar'.

Reply:
Added missing imports

Reply:
Which division do you want? Floor division (integer result) or true division (could be a float result)?  Future import of division will force the / operator from floor division to true division for integers.  However this will make it a bit unclear when somebody is looking at the code directly. If you want floor division in python 2 and 3 you can use the // operator. It will always perform floor division. If you want true division you could explicitly cast one of the two to a float and you will unambiguously get true division. It's not needed for functionality, but it would improve clarity without having to skip back to the imports to check.